.github/workflows/python-package.yml

```diff
@@ -35,6 +35,11 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Test with pytest
+    - name: Build the Docker image
       run: |
-        python -m pytest
+        docker build -t amazon/aws-glue-libs:glue_libs_3.0.0_image_01 .
```

Comment:
A more formal way to do this is to have [a separate job which runs inside a docker container](https://docs.github.com/en/actions/creating-actions/creating-a-docker-container-action). Since you are using the image directly (without installing any dependencies or adding additional layers to the image) this should be a relatively straightforward change to make.

Reply:
Although there are some "gotchas" with doing things that way. Specifically, you need to be root to perform actions like `actions/checkout@v2`, but all the glue dependencies are installed under the `glue_user`. You can see how we did this in Bridge Downstream here: https://github.com/Sage-Bionetworks/BridgeDownstream/blob/main/.github/workflows/upload-and-deploy.yaml#L92-L124